README.md

```diff
@@ -83,7 +83,7 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 - Build react app
   - run `npm run build`
 - Build docker image:
-  - run `docker build -t krispykopters/<unique tag> .`
+  - run `docker build . -t krispykopters/<unique tag> .`
```

Comment:
There is a dot at the end of the command already, adding another one would break the command right?

Reply:
Oh, but the first command does not work in my environment. It works after i add the dot. Does the first one work in yours? If yes, ommit this change.